build-script.md

```diff
@@ -0,0 +1,317 @@
+---
+layout: default
+title: Building KLEE and its Dependencies
+subtitle: The build infrastructure for KLEE
+slug: build-klee-script
+---
+
+# Building KLEE and its Dependencies
+
+KLEE is a symbolic execution framework that can be built with a multitude of different components (compiler infrastructure: LLVM 3.8 - ..., solvers: STP, Z3, MetaSMT) and run on a variety of systems (e.g., Linux (Ubuntu), Mac OS X, FreeBSD).
```

Comment:
Minor (if you're going over other proofreading changes): I would replace "LLVM 3.8 - ..." by "multiple LLVM versions", so that we don't have to update this when the versions supported change.
Also, "Mac OS X" is now "macOS"